.planning/quick/260728-umo-admin-coffee-shop-photo-reorder-add-sort/SUMMARY.md

```
@@ -0,0 +1,20 @@
+---
+status: complete
+id: 260728-umo
+slug: admin-coffee-shop-photo-reorder-add-sort
+---
+
+# Summary: Admin coffee shop photo reorder
+
+## Done
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)